.tekton/central-db-build.yaml

```diff
@@ -8,13 +8,10 @@ metadata:
     build.appstudio.redhat.com/pull_request_number: '{{pull_request_number}}'
     build.appstudio.redhat.com/target_branch: '{{target_branch}}'
     pipelinesascode.tekton.dev/max-keep-runs: "500"
-    # TODO(ROX-21073): re-enable for all PR branches
-    pipelinesascode.tekton.dev/on-cel-expression: |
-      (event == "push" && target_branch.matches("^(master|release-.*)$")) ||
-      (event == "pull_request" && (source_branch.matches("(konflux|appstudio|rhtap)") || body.pull_request.labels.exists(l, l.name == "konflux-build")))
+    pipelinesascode.tekton.dev/on-cel-expression: target_branch == "release-4.6" && event in ["push", "pull_request"]
```

Comment:
Can we make that CEL expression match also the regex `4\.6\..*` for branches? :)

Reply:
Based on the [pipelines as code doc](https://pipelinesascode.com/docs/guide/authoringprs/#matching-an-event-to-a-pipelinerun), it looks like a tag can be referenced in the `target_branch` as `refs/heads/<tag>`, so that would look like `refs/tags/4\.6\..?`

Reply:
I had to [crack open the pipelines-as-code code](https://github.com/openshift-pipelines/pipelines-as-code/blob/aaf045e8678e99957ef68fa40b3fd86dd6158530/pkg/matcher/cel.go#L27-L47) to understand what is going on here.

The `refs/tags` prefix is actually included in every `target_branch` value, so when I mixed `target_branch` values -- one with `refs/tags` and one without -- it caused the one _without_ `refs/tags` to _not_ match because the presence of `refs/tags` in the other `target_branch` stopped pipelines-as-code to _not_ strip `refs/tags`.  :dizzy_face:

Reply:
I needed two backslashes in when escaping the dots.  (I found this out by copying the CEL eval code and running it locally.)